frontend/packages/pipelines-plugin/src/components/quicksearch/PipelineQuickSearchVersionDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Dropdown, DropdownItem, DropdownToggle } from '@patternfly/react-core';
+import { CheckCircleIcon } from '@patternfly/react-icons';
+import { global_palette_green_500 as greenColor } from '@patternfly/react-tokens/dist/js/global_palette_green_500';
+import i18n from 'i18next';
+import { CatalogItem } from 'packages/console-dynamic-plugin-sdk/src';
+import { isSelectedVersionInstalled } from './pipeline-quicksearch-utils';
+
+interface PipelineQuickSearchVersionDropdownProps {
+  selectedVersion: string;
+  item: CatalogItem;
+  onChange: (key: string) => void;
+}
+
+const PipelineQuickSearchVersionDropdown: React.FC<PipelineQuickSearchVersionDropdownProps> = ({
+  item,
+  onChange,
+  selectedVersion,
+}) => {
+  const [isOpen, setOpen] = React.useState(false);
+  const toggleIsOpen = React.useCallback(() => setOpen((v) => !v), []);
+  const versions = item?.attributes?.versions ?? [];
+  const versionItems = versions.reduce((acc, { version, id }) => {
+    acc[id.toString()] =
+      id === item.data?.latestVersion?.id
+        ? `${version} ${i18n.t('pipelines-plugin~(latest)')}`
```

Comment:
Better to combine both version and latest in a sentence. And can we use `useTranslation` hook instead of `i18n`?
```suggestion
        ? t('pipelines-plugin~{{version}} (latest)', { version })
```